src/main/java/frc/robot/wrist/WristSubsystem.java

```diff
@@ -42,7 +41,7 @@ public double getAngle() {
 
   @Override
   public void periodic() {
-    SmartDashboard.putNumber("Wrist position", getAngle());
+    SmartDashboard.putNumber("Wrist/Position", getAngle());
     this.pid.setReference(
         WristSubsystem.GEARING.afterToBeforeGearing(goal.angle), CANSparkMax.ControlType.kPosition);
   }
```

Comment:
@saikiranra does calling `.setReference()` every tick instead of only when updated cause CAN congestion? does revlib optimize for this case?

Reply:
[conversation moved to discord](https://discord.com/channels/499769786780221472/499775401468362753/1033036022340145162)